.circleci/config.yml

```diff
@@ -2093,7 +2096,14 @@ workflows:
 
   scheduled-docker-publish:
     when:
-      equal: [ build_hourly, <<pipeline.schedule.name>> ]
+      and:
+        - or:
+            # Trigger on new commits
+          - equal: [ webhook, << pipeline.trigger_source >> ]
+            # Trigger on manual triggers if explicitly requested
+          - equal: [ true, << pipeline.parameters.main_dispatch >> ]
+        - not:
+            equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
```

Comment:
Will revert this trigger change before this PR is merged

Reply:
If the trigger is permissioned correctly then I don't see a reason to remove it. Seems useful to trigger it when necessary

Reply:
Changed this so that it runs on the hourly job (as before) and also can be triggered manually:
```
    when:
      - or:
        - equal: [ build_hourly, <<pipeline.schedule.name>> ]
        # Trigger on manual triggers if explicitly requested
        - equal: [ true, << pipeline.parameters.docker_publish_dispatch >> ]
```